dockerfiles/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+USER_ID=${LOCAL_USER_ID:-9001}
+
+useradd --shell /bin/bash -u $USER_ID -o -c "" developer
+usermod -aG docker developer
+sudo chmod 666 /var/run/docker.sock
+echo "developer ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+chown developer /home/developer
+chgrp developer /home/developer
+
+export HOME=/home/developer
+
+export LANG=en_US.UTF-8
+cd /home/developer/sdl/sdl_atf/build/bin/RemoteTestingAdapterServer
+ifconfig
+sudo LD_LIBRARY_PATH="/home/developer/sdl/3rd_party/lib:/home/developer/sdl/3rd_party/x86_64/lib:." -u developer "./RemoteTestingAdapterServer"
```

Comment:
Starting of `RemoteTestingAdapterServer` is implemented in https://raw.githubusercontent.com/LuxoftSDL/sdl_core_ci/impl/atf_remote_upd/scripts/atf_runner_remote.sh which is used in test jobs for remote connection:
 - Dockerfiles_u18_ATF_Smoke_TCP_Remote_P
 - Dockerfiles_u20_ATF_Smoke_TCP_Remote_P